data/tmpl/values/content3.html

```diff
@@ -32,7 +32,11 @@
             </label>
           </li>
         </ul>
-        <button class="button">Tell Me</button>
+        <button id="tell_me" class="button">Tell Me</button>
+        <section id="private_browsing" class="hidden">
```

Comment:
We should rethink this. I'm not sure exactly what the correct method might be for UX here. Maybe Matt can answer that sometime soon.